documentation/source/user_section/installation.rst

```diff
@@ -158,16 +158,11 @@ Windows subsystem for Linux (WSL) is available on Windows 10 and it makes it pos
            source ${SCT_DIR}/python/etc/profile.d/conda.sh
            conda activate venv_sct
 
-   Install wxPython using conda:
-
-   .. code-block:: sh
-
-           yes | conda install -c anaconda wxpython
-
-   Install FSLeyes using conda-forge:
+   Set the channel priority to strict (`as recommended by conda <https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-channels.html#strict-channel-priority>`_), then install FSLeyes using conda-forge:
 
    .. code-block:: sh
 
+           conda config --set channel_priority strict
            yes | conda install -c conda-forge fsleyes
```

Comment:
It's probably better to use ` -y, --yes             Do not ask for confirmation` since `conda` supports it.

Reply:
Now that you mention it, `yes |` is still used all throughout the installation docs. We forgot to update as part of #3102. :sweat_smile: